python/lsst/obs/base/_instrument.py

```diff
@@ -482,21 +491,27 @@ class attribute for curated calibrations corresponding to the
         # class.
         calib_class = datasetType.storageClass.pytype
         if not hasattr(calib_class, "readText"):
-            raise ValueError(
-                f"Curated calibration {datasetType.name} is using a class "
-                f"{get_full_type_name(calib_class)} that lacks a readText class method"
-            )
+            # Let's try the default calib class.  All curated
+            # calibrations should be subclasses of that.
+            calib_class = doImport('lsst.ip.isr.IsrCalib')
+
         calib_class = cast(Type[CuratedCalibration], calib_class)
 
         # Read calibs, registering a new run for each CALIBDATE as needed.
         # We try to avoid registering runs multiple times as an optimization
         # by putting them in the ``runs`` set that was passed in.
         camera = self.getCamera()
-        calibsDict = read_all(calibPath, camera, calib_class)[0]  # second return is calib type
+        filters = list(self.filterDefinitions.physical_to_band.keys())
+        if datasetType.name in StandardCuratedCalibrationDatasetTypes.keys():
```

Comment:
It took me removing this to remember the problem: the tests attempt to ingest a `testCalib` that doesn't exist in the `StandardCuratedCalibrationDatasetTypes`.  This was my solution to avoid having that test break: if the `datasetType` isn't standard, but it has a proper set of dimensions let it continue.  I've added a comment and log warning for this fall back case.